src/cli/rustup_mode.rs

```diff
@@ -1334,11 +1335,22 @@ fn target_remove(cfg: &Cfg, m: &ArgMatches) -> Result<utils::ExitCode> {
     let distributable = DistributableToolchain::try_from(&toolchain)?;
 
     for target in m.get_many::<String>("target").unwrap() {
-        let new_component = Component::new(
-            "rust-std".to_string(),
-            Some(TargetTriple::new(target)),
-            false,
-        );
+        let target = TargetTriple::new(target);
+        let default_target = cfg.get_default_host_triple().ok();
```

Comment:
Why not return any error from `get_default_host_triple()` up the stack? Same for `distributable.components()`.

Reply:
@djc Nice question! I'm a bit afraid of introducing unnecessary points of failure here since I'm just enhancing the error message, not performing some critical logic. However I can change that if you'd like.

Reply:
Changed in 3810964e1483e75e35dd3d7731bfb22887709b43.